src/cirrus/plugins/publishers/documentation.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+"""
+_documentation_
```

Comment:
Is there a reason that the module name doesn't match the PLUGGAGE_OBJECT_NAME - i.e. why not call this module `doc_file_server.py`? documentation is not very descriptive of what it does.